### PR TITLE
Added two new fields to content image form

### PR DESCRIPTION
**Pull request by guillelopez**

### CODE REVIEW

File: app/views/clear_cms/contents/_form.html.erb [hunk 1]
```diff
@@ -133,6 +133,8 @@ var linkedFiltersType = {
                     <%= caf.input :description, :as=>:text, :input_html=>{:rows=>1} %>
                     <%= caf.input :credit %>
                     <%= caf.input :tags %>
+                    <%= caf.input :css_text_color %>
```
Comment (stephenhmarsh) [inline app/views/clear_cms/contents/_form.html.erb]: Does this work without declaring ``field :css_text_color`` and ``field :css_background_position`` on ContentAsset model?

Comment (guillelopez) [inline app/views/clear_cms/contents/_form.html.erb]: I added those to https://github.com/assembledbrands/theline/blob/changes-to-hp-chapters-pagination/app/models/content_asset_decorator.rb. But you are right it makes more sense to add it here. I will change that.

Comment (guillelopez) [inline app/views/clear_cms/contents/_form.html.erb]: I will be adding as well this change: https://github.com/assembledbrands/theline/blob/changes-to-hp-chapters-pagination/app/models/content_asset_decorator.rb#L3

Comment (guillelopez) [inline app/views/clear_cms/contents/_form.html.erb]: Ok, done.

**Pull request merged**